src/AaveV3_Pol_CapsUpdate_20230608/CapsUpdate.md

```diff
@@ -0,0 +1,48 @@
+---
+title: [ARFC] Increase MaticX supply cap
+author: Alice Rozengarden (@Rozengarden - Aave-chan initiative)
+discussions: https://governance.aave.com/t/arfc-increase-maticx-supply-cap/14341
```

Comment:
```suggestion
title: "[ARFC] Increase MaticX supply cap"
author: "Alice Rozengarden (@Rozengarden - Aave-chan initiative)"
discussions: "https://governance.aave.com/t/arfc-increase-maticx-supply-cap/14341"
```

not quite sure why it files, but probably better to just escape the strings - will update the template